source/includes/wp-api-v3/_order-refunds.md

```diff
@@ -70,13 +72,41 @@ curl -X POST https://example.com/wp-json/wc/v3/orders/723/refunds \
 	-u consumer_key:consumer_secret \
 	-H "Content-Type: application/json" \
 	-d '{
-  "amount": "10"
+  "order_refund": {
+    "amount": 30,
+    "line_items": [
+      {
+        "id": "111",
+        "refund_amount": 10,
+        "refund_tax": [
+          {
+            "id" "222",
+            "refund_amount": 20
+          }
+        ]
+      }
+    ]
+  }
 }'
 ```
 
 ```javascript
 const data = {
-  amount: "10"
+    "order_refund": {
```

Comment:
This is an incorrect syntax for JavaScript

Reply:
Yep, no quotes needed!